docs/known_issues.md

```diff
@@ -14,15 +14,24 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## Compatibility with Rails form helpers
 
 ViewComponent [isn't compatible](https://github.com/viewcomponent/view_component/issues/241) with `form_for` helpers by default.
 
-## Inconsistent controller rendering behavior between Rails versions
+Passing a form object reference (usually `f`) to a view component works for simple cases, like `f.text_field :name`.
+Content will be ill-ordered or duplicated in complex cases (such as passing blocks to form helpers or when nesting components).
```

Comment:
This may be unhelpful, but — is this limited to form helpers? I feel like I remember seeing issues with other block capturing helpers, and `link_to` is similarly mentioned in one of the issues: https://github.com/ViewComponent/view_component/issues/241#issuecomment-905233222

Does that help clarify the difference between the "simple" and "complex" cases?

Reply:
Turbo helpers as well.

Reply:
Yes, the compatibility issues affect numerous Rails helpers. Another one that hasn't been mentioned yet is `content_for`, and I think `content_tag` too.

Reply:
Shall we mention the other affected helpers as well? Maybe within an additional section?

```markdown
## Other Rails helpers with compatibility issues

Similar to the form helpers (see above), the following helpers have compatibility issues:

- `content_for` --> Set directly within a Rails view
- `link_to`(?) --> ???
- `content_tag` --> Use xzy instead

To keep this PR small, I would like to do this in another PR

Reply:
Yeah another PR sounds like a good idea.

For both `link_to` and `content_tag`, the best suggestion is probably to just write the HTML by hand 😢